src/components/shareButton/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Button } from 'antd';
+import { SizeType } from 'antd/es/config-provider/SizeContext';
+import ShareMenu from '../shareMenu';
+import ShareWhite from '../../assets/images/sharewhite-small.png';
+
+const StyledShareButton = styled(Button)`
+  && {
```

Comment:
:question: Are the `&&`s necessary? This seems to be an equivalent of `!important` and we generally want to use it sparingly

Reply:
Took another look, they're not needed - will keep that in mind!